src/RSEF/object_creator/create_downloadedObj.py

```diff
@@ -192,12 +192,131 @@ def pdf_to_downloaded_obj(pdf,output_dir):
     arxiv = extract_arxivID(resp_jsn)
     return DownloadedObj(title=titL,doi=doi,arxiv=arxiv,file_name="",file_path=pdf)
 
+# Download papers from a json containing title, doi, primary location (PDF's URL)
+def json_to_downloaded_obj(json_data, output_dir):
+    if not os.path.exists(json_data):
+        print(f"Error: JSON file '{json_data}' does not exist.")
+        return None
+    try:
+        # Clear the content of the file downloaded_metadata.json
+        downloaded_metadata_path = os.path.join(output_dir, "downloaded_metadata.json")
```

Comment:
same as above. Have a constants file with the default path/name of the downloaded metadata json file